google/tie
# Fix AttributeError Regex Bug

* Adjusted AttributeError regex for when error names an attribute that includes non-word characters

## To test:
* Before importing changes, try using a `length` method or property for a string
* Note that an error is thrown because `found` variable is null
* Import changes and now no error should be thrown
